tests/my_test_Inference_RAxML.py

```diff
@@ -0,0 +1,264 @@
+from pathlib import Path
+import subprocess
+import pytest
+import random
+import os
+
+from Bio import Phylo
+from Bio.Phylo.Consensus import _BitString
+
+from mevolib.inference import _RAxML as Rax
+from mevolib._utils import NUMCORES
+
+
+class MockStdOut:
+    def __init__(self, tree_infile_path, tree_outfile_path, info_infile_path, info_outfile_path):
+        with open(tree_infile_path, "r") as fin:
+            self.stdout_tree = fin.read()
+        with open(tree_outfile_path, "w") as fin:
+            fin.write(self.stdout_tree)
+        with open(info_infile_path, "r") as fin:
+            self.stdout_info = fin.read()
+        with open(info_outfile_path, "w") as fin:
+            fin.write(self.stdout_info)
```

Comment:
I do not think you need to copy the files, you can just return its content directly when needed.

Reply:
The subprocess.run() method does it, and the RAxML get_results functions needs to read the file ended at n (random number) placed at raxml_tmp_dir, so it's needed.